docs/source/tracking.rst

```diff
@@ -1028,7 +1035,7 @@ to access Google Cloud Storage; MLflow does not declare a dependency on this pac
 
 You may set some MLflow environment variables to troubleshoot GCS read-timeouts (eg. due to slow transfer speeds) using the following variables:
 
-- ``MLFLOW_GCS_DEFAULT_TIMEOUT`` - Sets the standard timeout for transfer operations in seconds (Default: 60). Use -1 for indefinite timeout.
+- ``MLFLOW_UPLOAD_DOWNLOAD_TIMEOUT`` - Sets the standard timeout for transfer operations in seconds (Default: 60). Use -1 for indefinite timeout.
```

Comment:
```suggestion
- ``MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT`` - Sets the standard timeout for transfer operations in seconds (Default: 60). Use -1 for indefinite timeout.
```

Reply:
Is there a use case for setting different upload / download timeouts, i.e. splitting this into two vars? I think one is fine, but just checking